### DISCUSSION TITLE
[Help] Using `>>` to transfer text to an specific line or under a specific header

### DISCUSSION THREAD

**Discussion by FelipeRearden** [Taitava/obsidian-shellcommands#82] [Q&A] [answered]
Hello @Taitava !!!!

Do you know if we have an command to append text to a file at an specific line ?

My user case: I wanna append text under an specific header in the file INP.md 

I searched but I didn't find anything direct to this case :(

I could use an Obsidian URI to do this but the problem here is that it will open the destination note :(

Have a great day!

**Comment by Taitava** [comment, marked as answer]
Hi @FelipeRearden !

How about: https://stackoverflow.com/a/22159219/2754026
I do not know if this helps or not. 

See also the comment in that answer:
> How to make this change in-place? Is there a better solution than my dirty hack? `awk '{if (/^After\=/) {$0=$0 " postgresql.service"}; print}' jetty.service > jetty.service.tmp && mv -f jetty.service.tmp jetty.service`

And there the question seems to be how to insert at the end of a certain line, not below it, but I guess a simple `\n` should help you put the inserted text on a new line?

**Reply by FelipeRearden** [reply]
@Taitava !!!!

Thank you very much for one more help. I think the `awk` is the best way to achieve this. 

Let me just study a little bit this stack overflow post to understand before doing something. I get back to you when I finish this. 

But I think that you found exactly  what I was looking for :)

Have a great day!

**Reply by Taitava** [reply]
Good! Let me know if you need something different! :slightly_smiling_face:

**Reply by FelipeRearden** [reply]
> Good! Let me know if you need something different! :slightly_smiling_face: 

Thank you very much @Taitava !!! you are super !!

I'm running a little bit late today but tomorrow I will try to use the `awk` command :)

Let keep talking :)

**Reply by FelipeRearden** [reply]
Just to let you know that the `awk` works!!!! forgot to tell you :)

Thanks @Taitava !!!!!

**Reply by Taitava** [reply]
Good! 🌞
